src/components/DepositWidget/Row.tsx

```diff
@@ -136,13 +136,17 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
           {isWeth ? (
             <ul>
               <li title={ethBalance ? formatAmountFull({ amount: ethBalance, precision: decimals }) : undefined}>
-                {ethBalance ? formatSmart(ethBalance, decimals) : '-'} ETH{' '}
+                <span>
+                  {ethBalance ? formatSmart(ethBalance, decimals) : '-'} <span>ETH</span>
```

Comment:
wouldn't `word-break: break-word;` here also work? i.e no need for additional `span` but also whatever if this works, yolo

Reply:
alignment becomes harder to fit both lines uniformally
![localhost_8080_wallet (4)](https://user-images.githubusercontent.com/5121491/89995845-bcda9900-dc92-11ea-801e-1a22d8b01a02.png)

You can play around with it if you want, though

Reply:
ew no i like ur approach 1000x